src/test/scala/DateParserAppSuite.scala

```diff
@@ -0,0 +1,32 @@
+import org.scalatest.freespec.AnyFreeSpec
+import wvlet.log.{LogLevel, Logger}
+
+import java.text.ParseException
+
+class DateParserAppSuite extends AnyFreeSpec {
+  Logger.setDefaultLogLevel(LogLevel.DEBUG)
+  val date_parser = new DateParserApp()
+  "DateParserAppSuite" - {
+    "date_parser" - {
+      "should parse good date with good format " in {
+        assert(date_parser.parse_date("yyyy-MM-dd", "2020-12-12").toString == "Sat Dec 12 00:00:00 UTC 2020")
```

Comment:
Now tests fail locally (instead of UTC I have CET), how to compare the date better?

Reply:
When you switch to Java Time you will have to provide a time zone to the formatter, so this shouldn't be an issue.